config/boot.rb

```diff
@@ -1,3 +1,15 @@
 ENV['BUNDLE_GEMFILE'] ||= File.expand_path('../../Gemfile', __FILE__)
 
 require 'bundler/setup' # Set up gems listed in the Gemfile.
+
+require 'rubygems'
+require 'rails/commands/server'
```

Comment:
Prefer double-quoted strings unless you need single quotes to avoid extra backslashes for escaping.